scripts/config/templates/nginx.conf.erb

```diff
@@ -468,6 +468,11 @@ http {
     proxy_pass <%= ssr_frontend_host %>/business$is_args$args;
   }
 
+  # redirect to speedy home page in ssr-frontend
+  location = "/" {
```

Comment:
feel free to merge this first, but you might need to do an empty string or a "/", or even better redirect the "/" to the empty string. I just don't know how the location is handled for an empty path like this